### PR TITLE
Update ScalaTest to 2.0

### PR DESCRIPTION
**Pull request by alexbergeron**

### CODE REVIEW

File: mry-core/src/it/scala/com/wajam/mry/storage/mysql/FeederTestHelper.scala [hunk 1]
```diff
@@ -3,6 +3,7 @@ package com.wajam.mry.storage.mysql
 import com.wajam.spnl.feeder.Feeder
 import com.wajam.commons.Closable
 import com.wajam.spnl.feeder.Feeder.FeederData
+import scala.language.implicitConversions
```
Comment (mmimeault) [inline mry-core/src/it/scala/com/wajam/mry/storage/mysql/FeederTestHelper.scala:6]: why do we need that now ?

Comment (alexbergeron) [inline mry-core/src/it/scala/com/wajam/mry/storage/mysql/FeederTestHelper.scala:6]: Compiler warning

File: mry-core/src/test/scala/com/wajam/mry/api/protobuf/TestMryCodec.scala [hunk 2]
```diff
@@ -21,7 +21,7 @@ class TestMryCodec extends FunSuite with BeforeAndAfter with ShouldMatchers {
     val bytes = codec.encode(n)
     val n2 = codec.decode(bytes)
 
-    n2 should equal(null)
+    n2 should === (null)
```
Comment (mmimeault) [inline mry-core/src/test/scala/com/wajam/mry/api/protobuf/TestMryCodec.scala:24]: equal not supported anymore ??
why (null)... could be `n2 should === null` ?

Comment (alexbergeron) [inline mry-core/src/test/scala/com/wajam/mry/api/protobuf/TestMryCodec.scala:24]: equal is still supported, but this didn't compiled for some reason.

I kept the parens as suggested by http://www.scalatest.org/user_guide/using_matchers#thosePeskyParens

> Although you don't always need them, you can choose to always put parentheses around right-hand values

**Comment by mmimeault** [comment]
+1

**Pull request merged**